includes/rcon.sh

```diff
@@ -1,57 +1,56 @@
 # shellcheck disable=SC2148
 
+function get_time() {
```

Comment:
Why is this a function that gets called multiple times and always has to calculate the date? Why not defined once, use once the resources and save it to memory. What am i missing? Why is this better?

Reply:
This is run every time for each line because each line is broadcasted at a different time, and each line needs to have an accurate timestamp for when it was broadcasted.

Take backups for example. It takes 10+ seconds to run, so calculating unique timestamps for each broadcast will accurately show when the it starts and ends when broadcasted in the game.